### PR TITLE
meta: rework how to manage issues

### PR DESCRIPTION
**Pull request by fzn0x**
This helps manage issues easier, and keeps user to follow the format and understand what to do if there is a bug they want to report, this is also aims to prevent an issue being moved to discussions.

### CODE REVIEW

**Comment by fzn0x** [comment]
Example of issues config result: https://github.com/nodejs/node/issues/new/choose
https://github.com/nodejs/node/blob/main/.github/ISSUE_TEMPLATE/config.yml

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Supports and troubleshooting
+    url: https://github.com/sequelize/sequelize/discussions
+    about: This issue tracker is not for bug reports which requires a SSCCE. Please refer to the bug report issue template.
```
Comment (ephys) [inline .github/ISSUE_TEMPLATE/config.yml:5]: The issue tracker is for all bug reports and feature requests. If a bug report can't be replicated in a SSCCE, it should still go in the tracker

I propose to rename the link "Help & Questions" and to change its description to "The issue tracker is for bug reports and feature requests. Use GitHub Discussions for question and help requests"?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  - name: SSCCE
```
Comment (ephys) [inline .github/ISSUE_TEMPLATE/config.yml:6]: Linking to the SSCCE from here isn't very useful. It's something they only need if they're reporting a bug and it's already part of the bug report template

**Review by ephys — CHANGES_REQUESTED** [review]
This would be a great change. I didn't know github supported this! I've been looking for something like this for a long time

I think the SSCCE link isn't necessary, but here are a few links we could add:

- A link about website documentation issues that point to the website's repository
- A link about sequelize-cli issues that point to the CLI repository
- A link about reporting Security Vulnerabilities that point to https://github.com/sequelize/sequelize/blob/main/SECURITY.md (looks like GitHub has a special option for that too)

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (fzn0x) [reply .github/ISSUE_TEMPLATE/config.yml:6]: Thank you for the inputs!

**Comment by fzn0x** [comment]
I will make sure I make it fancier 😉

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (fzn0x) [reply .github/ISSUE_TEMPLATE/config.yml:5]: That's better

**Comment by WikiRik** [comment]
And for a follow up PR; we might want to look into [issue forms](https://github.blog/changelog/2021-06-23-issues-forms-beta-for-public-repositories/) as well.

**Comment by fzn0x** [comment]
> And for a follow up PR; we might want to look into [issue forms](https://github.blog/changelog/2021-06-23-issues-forms-beta-for-public-repositories/) as well.

Looks good! Lets make Sequelize better

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
+  - name: ❓ Help & Questions
```
Comment (ephys) [inline .github/ISSUE_TEMPLATE/config.yml:3]: The Help & Question page already uses this emoji, let's keep it unified:

```suggestion
  - name: 💬 Help & Questions
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    url: https://github.com/sequelize/sequelize/discussions
+    about: The issue tracker is for bug reports and feature requests. Use GitHub Discussions for question and help requests.
+  - name: 🔧 Having issues with Sequelize CLI?
+    url: https://github.com/sequelize/cli/issues
+    about: The issue tracker is for Sequelize CLI issues.
+  - name: 📗 Having issues with Sequelize documentation?
+    url: https://github.com/sequelize/website/issues
+    about: The issue tracker is for Sequelize documentation issues.
```
Comment (ephys) [inline .github/ISSUE_TEMPLATE/config.yml:11]:
```suggestion
  - name: 📗 Found an issue in our documentation?
    url: https://github.com/sequelize/website/issues
    about: Please file an issue in the Sequelize website repository.
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
+  - name: ❓ Help & Questions
+    url: https://github.com/sequelize/sequelize/discussions
+    about: The issue tracker is for bug reports and feature requests. Use GitHub Discussions for question and help requests.
+  - name: 🔧 Having issues with Sequelize CLI?
+    url: https://github.com/sequelize/cli/issues
+    about: The issue tracker is for Sequelize CLI issues.
```
Comment (ephys) [inline .github/ISSUE_TEMPLATE/config.yml:8]:
```suggestion
    about: Please file an issue in the Sequelize CLI repository.
```

**Review by ephys — CHANGES_REQUESTED** [review]
Misc propositions

**Review by ephys — APPROVED** [review]

**Comment by fzn0x** [comment]
Screenshot 2022-11-19 212516

It's looking pretty now

File: .github/ISSUE_TEMPLATE/docs_issue.md [hunk 6]
```diff
@@ -1,31 +0,0 @@
----
```
Comment (WikiRik) [inline .github/ISSUE_TEMPLATE/docs_issue.md:1]: @fzn0x I believe that the website repo is missing issue templates. Can you add this there? Maybe with the config as well?

Comment (fzn0x) [reply .github/ISSUE_TEMPLATE/docs_issue.md:1]: Sure @WikiRik

Comment (fzn0x) [reply .github/ISSUE_TEMPLATE/docs_issue.md:1]: Track https://github.com/sequelize/website/pull/349

**Pull request merged**